modules/node_modules/@colony/purser-software/class.js

```diff
@@ -55,17 +55,10 @@ export default class SoftwareWallet {
 
   privateKey: string;
 
-  publicKey: string;
-
   originalMnemonic: string;
 
   derivationPath: string;
 
-  /*
-   * Encrypted JSON Keystore
-   */
-  keystore: string;
```

Comment:
Flow showed me that these had to be removed, because they conflicted with the getters (which are typed appropriately).